### ISSUE TITLE
Group tabs rename themselves to "Group", lose their functionality and I have to click the bookmark again to make a new group tab and then rename it.

### ISSUE THREAD

**Issue by tstftw** [piroor/treestyletab#1976]
## Short description
I have a bookmark for the group tabs (moz-extension://37d71e54-6d9d-460f-a810-0ae85880916a/resources/group-tab.html?title=Folder&temporary=false)
about:treestyletab-group?title=foobar gives me an error: "Hmm. That address doesn’t look right. Please check that the URL is correct and try again"

Today I restarted Firefox and all the folder/group tabs that are roots of each tree have the title "Group", but when I click on them, I don't see the regular page with the group name on top and the tabs in the tree listed on the page. Instead I see a white page with nothing in it.

Also, quite often, when I restart Firefox or just drag a subtree around, some trees lose their position and appear as root trees. Quite annoying, but I guess that's for another bug report.

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. Use group tabs by clicking from a bookmark (or pasting the URL in the address bar).
 4. Give a group tab a name.
 4. Restart Firefox.

I found this extremely useful, because I have several projects going and and each one was under a group/folder tab with the name of the project (e.g., Hardware, Study, Security, Firefox, Music, Torrent).

## Expected result

Group tabs work as they have worked before today. Their behave as normal tabs, but retain their name 

## Actual result

Group tabs lose their names, instead show "Group". The group tabs lose their functionality, i.e. they don't know their name and the subtrees on the page (i.e., when I select the group tab).

## Environment

 * Platform (OS): Fedora 27
 * Version of Firefox: 61.0
 * Version (or revision) of Tree Style Tab: 2.4.24

EDIT: The problem also occured just now when I selected the first of my group tabs and opened a new tab by clicking CTRL+T.

EDIT: I added a screenshot. https://i.imgur.com/RdOEdji.png The tab was called "Misc" before it became "Group". Also, on "Group" and "Music" there's a weird favicon for some reason.

EDIT: Now it happens just by clicking any group tabs. Now all my group tabs are called "Group". 
https://i.imgur.com/Ko00FBO.png This should that after clicking on each group tab, only "Music" remained funcitonal (and actually shows the page with the name and the opened tabs (in this case only 1).

Any workaround or solution? Should I just use make my own html files with a certain title and place them in place of the group tabs?

EDIT: A workaround I found was to make several html files with title only. It doesn't let me easily rename the tabs and I can't view the whole list of opened tabs from the page, but it's enough for me. I'd still like an actual solution, though. Thanks!

**Comment by piroor** [comment]
Is this reproducible with the latest development build?: https://github.com/piroor/treestyletab/blob/master/README.md#development-builds

**Comment by tstftw** [comment]
Yes.

I installed TST 2.4.24.7365
Replaced my manual folder tabs that are just empty local html pages with only a title with the moz-extension://37d71e54-6d9d-460f-a810-0ae85880916a/resources/group-tab.html?title=Folder&temporary=false link and renamed them.
Restarted Firefox.
Clicked on some of the folder tabs and some of them changed to "Group", like before. Some of them still work, but some of them are just "Group". The "Group" tabs show no URL in the address bar.

I'll revert back to the locla html files.

**Comment by tstftw** [comment]
So far it works. I installed the new build a few minutes ago. My previous local files that I used for group tabs disappeared (all but the first one) and I had to rearrage most of my tabs and restart Firefox to clear an inconsistency between the actual tree structure of the tabs and the way it is shown (which happens every few days). But the problem with the self-renaming group tabs is solved, I think. Thanks!
